docs/zh/connector-v2/sink/Doris.md

```diff
@@ -147,6 +147,13 @@ CREATE TABLE IF NOT EXISTS `${database}`.`${table_name}`
 
 支持的格式包括 CSV 和 JSON。
 
+## 调优指南
+
+适当增加`sink.buffer-size`和`doris.batch.size`的值可以提高写性能。< br >
+在流模式下，如果`doris.batch.size`和`checkpoint.interval`都配置为较大的值，最后到达的数据可能会有较大的延迟(延迟的时间就是检查点间隔的时间)。< br >
```

Comment:
@dailai Please modify `< br >` to `<br>`

Reply:
Done